docs/notebooks/annoytutorial.ipynb

```diff
@@ -0,0 +1,356 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is Annoy?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Annoy](https://github.com/spotify/annoy \"Link to annoy repo\") is an open source library to search for points in space that are close to a given query point. It also creates large read-only file-based data structures that are mmapped into memory so that many processes may share the same data. For our purpose, it is used to find similarity between words or documents in a vector space. [See the tutorial on similarity queries for more information on them](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/Similarity_Queries.ipynb).\n",
+    "\n",
+    "[There are benefits](https://github.com/spotify/annoy#background) of using annoy over the pre-existing method of making similarity queries through brute force in gensim."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting Started"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First thing to do is to install annoy, by running the following in the command line:\n",
+    "\n",
+    "`sudo pip install annoy`\n",
+    "\n",
+    "And then set up the logger: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# import modules & set up logging\n",
+    "import logging\n",
+    "logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Making a Similarity Query"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the following examples, we'll use the Lee Corpus (which you already have if you've installed gensim)\n",
+    "\n",
+    "See the [Word2Vec tutorial](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/word2vec.ipynb) for how to initialize and save this model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Word2Vec(vocab=806, size=100, alpha=0.025)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Load the model\n",
+    "import gensim\n",
+    "model = gensim.models.Word2Vec.load('/tmp/leemodel')\n",
+    "print model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using the SimilarityIndex class\n",
+    "An instance of `SimilarityIndex` needs to be created in order to use Annoy in gensim. The `SimilarityIndex` class is located in `gensim/similarities/index`\n",
```

Comment:
If it's meant to be a filesystem path, use extension too (and perhaps even hyperlink).